vendor/lansitec/profile-as923.yaml

```diff
@@ -0,0 +1,48 @@
+vendorProfileID: 0

```

Comment:
Could you make sure all the profiles have unique `vendorProfileID`'s? I can see you have 7 so number them from 0 to 6 for example.

Reply:
The vendorProfileID has been modified from 0 to 6

Reply:
Thank you for that 👍 By the way, you can change multiple files in 1 commit. This helps to compact the history.

Reply:
Thank you for your suggestion